### PR TITLE
Math magicians: components

### PR DESCRIPTION
**Pull request by nilab-khwaja**
- Delete all the boilerplate from Create React App (text, images, styles).
- Create a new directory called components.
- Inside components, create a new calculator.js file.
- Implement the design only (JSX and CSS)
- Use functional components.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ root.render(
   </React.StrictMode>,
 );
 
-// If you want to start measuring performance in your app, pass a function
-// to log results (for example: reportWebVitals(console.log))
-// or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
-reportWebVitals();
+// // If you want to start measuring performance in your app, pass a function
```
Comment (GzimAsani) [inline src/index.js:14]: - [ ] All the boilerplate created by the CRA need to be deleted, kindly do this:
image

File: src/components/calculator.jsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      {/* numbers */}
+
+      <div className="zero">0</div>
+      <div>.</div>
+      <div className="orange">=</div>
+    </div>
+  </>
+);
+
+export default Calculator;
```
Comment (GzimAsani) [inline src/components/calculator.jsx:36]: - [ ] According to the project requirements the Calculator file needs to have at least 1 child element: 
image

Comment (GzimAsani) [reply src/components/calculator.jsx:36]: - [ ] You need to adjust the design in order to match the given design:

Your calculator:
image

Given design calculator:
image

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello, @nilab-khwaja            :wave:   
 Good job so far,  done There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job with creating a Calculator component t :+1: 

 ## Required Changes 

- [ ] Check the comments below :arrow_double_down: 

 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N/A



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/calculator.jsx (hunk 2 above)
Comment (nilab-khwaja) [reply src/components/calculator.jsx:36]: Hi @GzimAsani  
 I have already add a child to the component folder component.jsx , kindly tell me what do you mean by other child component?

Comment (GzimAsani) [reply src/components/calculator.jsx:36]: Coponent.jsx needs to have at least 1 child component

**Review by Rustamxon7 — APPROVED** [review]
Hi @nilab-khwaja, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Developer-512px-10

## Highlights
1. No Linters errors 💯
2. All requirements are complete 💯
3. Well-implemented code 💯 

## Optional suggestions
No optional suggestions :)

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**